packages/manager/apps/dedicated/client/app/dedicatedUniverseComponents/duration-iso8601/index.js

```diff
@@ -0,0 +1,13 @@
+import angular from 'angular';
+import translate from 'angular-translate';
+import ngTranslateAsyncLoader from '@ovh-ux/ng-translate-async-loader';
+import durationISO8601Filter from './duration-iso8601.filter';
+
+const moduleName = 'durationISO8601';
+
+angular
+  .module(moduleName, [ngTranslateAsyncLoader, translate])
+  .filter('durationISO8601', durationISO8601Filter)
```

Comment:
This filter could be set in this package: https://github.com/ovh/manager/tree/master/packages/manager/modules/filters